apps/server/test/lexical/server/code_intelligence/completion/translations/struct_test.exs

```diff
@@ -37,7 +37,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.StructTest do
       assert completion.kind == :struct
     end
 
-    test "should complete module aliases after %", %{project: project} do
+    test "should complete module aliases and ends with {} after %", %{project: project} do
```

Comment:
what about a test for not adding curlies when it already ends with {} ?

Reply:
o, there is already a test to do that; I need to reset this description.